app/controllers/v0/feature_toggles_controller.rb

```diff
@@ -12,16 +12,26 @@ def index
 
         features = features_params.collect do |feature_name|
           underscored_feature_name = feature_name.underscore
-          { name: feature_name, value: Flipper.enabled?(underscored_feature_name, actor(underscored_feature_name)) }
+          actor_type = FLIPPER_FEATURE_CONFIG['features'].dig(feature_name, 'actor_type')
+
+          { name: feature_name, value: Flipper.enabled?(underscored_feature_name, actor(actor_type)) }
         end
       else
         features = []
+
+        # returning both camel and snakecase for uniformity on FE
+        FLIPPER_FEATURE_CONFIG['features'].collect do |feature_name, values|
+          features << { name: feature_name.camelize(:lower),
+                        value: Flipper.enabled?(feature_name, actor(values['actor_type'])) }
+          features << { name: feature_name, value: Flipper.enabled?(feature_name, actor(values['actor_type'])) }
+        end
```

Comment:
Instead of returning both, could we base it on the `X-Key-Inflection` header?

Reply:
it doesn't look like the [FE code](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/utilities/feature-toggles/flipper-client.js#L8-L10) currently sets that header, @timwright12 does that seem like a reasonable thing to include?

Reply:
They won't have to include it but the feature mappings will need to be updated because they include both snack case and camelCase arbitrarily.

Reply:
@timwright12 confirmed that the FE doesn't currently set the header